### ISSUE TITLE
It is not possible to put emoji in labels for transactions

### ISSUE THREAD

**Issue by Urban-Hacker** [sparrowwallet/sparrow#1123]
While I was messing around with Sparrow I discovered you cannot set emoji in labels for transactions. While this might seem frivolous, it is well known that humans are better at distinguishing pictures than text and I could see it very useful to use emoji to categorize transactions: Eg: 👩‍⚕️ when you pay your doctor and ⛏️ for mining transaction.

During my tests I noticed that you can set really interesting character set such as "box drawing characters" and unicode symbols

![image](https://github.com/sparrowwallet/sparrow/assets/92275709/e6ec3177-0ac0-4ab5-af9a-ac37e31671a0)

**Comment by craigraw** [comment]
Yes, any UTF8 character is permissible: https://www.fileformat.info/info/charset/UTF-8/list.htm

Wrt emojis, there would need to be a particular emoji set chosen, and changing of all the relevant textfields and labels to handle this emoji set. I'm not convinced the additional dependencies and complexity are worth it however. Noting this project for JavaFX emoji support: https://github.com/pavlobu/emoji-text-flow-javafx
